examples/with-gorm/openapi.generated.yaml

```diff
@@ -147,35 +147,72 @@ paths:
                                 type: string
 components:
     schemas:
-        gin.H:
-            type: object
-            additionalProperties: {}
-        gorm.DeletedAt:
-            type: object
-            properties:
-                Time:
-                    $ref: '#/components/schemas/time.Time'
-                Valid:
-                    type: boolean
-        gorm.Model:
+        "":
```

Comment:
This doesn't look right to me

Reply:
This has now been updated, it was better but I have now added a fallback back if no binding tags are used.